fastlane/lib/fastlane/actions/gym.rb

```diff
@@ -60,15 +60,26 @@ def self.run(values)
           else
             self.show_xcode_9_warning
           end
+        elsif Actions.lane_context[SharedValues::SIGH_PROFILE_PATHS]
```

Comment:
I am not sure how these changes got added. My intention is to add change only to  [environment_printer.rb](https://github.com/fastlane/fastlane/pull/10284/files#diff-904c2f45a1357fe85cc1c98aeb122e08)